baseplate/frameworks/queue_consumer/kombu.py

```diff
@@ -241,3 +345,127 @@ def build_message_handler(self) -> KombuMessageHandler:
+        self.name = name
+        self.handler_fn = handler_fn
+        self.error_handler_fn = error_handler_fn
+        self.health_check_fn = health_check_fn
+        self.serializer = serializer
+        self.batch_size = batch_size
+        self.batch_timeout = batch_timeout
+
+    @classmethod
+    def new(
+        cls,
+        baseplate: Baseplate,
+        exchange: kombu.Exchange,
+        connection: kombu.Connection,
+        queue_name: str,
+        routing_keys: Sequence[str],
+        handler_fn: BatchHandler,
+        error_handler_fn: Optional[BatchErrorHandler] = None,
+        health_check_fn: Optional[HealthcheckCallback] = None,
+        serializer: Optional[KombuSerializer] = None,
+        batch_size: int = 1,
+        batch_timeout: timedelta = timedelta(0, 1, 0, 0, 0, 0, 0),
+    ) -> "KombuBatchQueueConsumerFactory":
+        """Return a new `KombuBatchQueueConsumerFactory`.
+
+        This method will create the :py:class:`~kombu.Queue` s for you and is
+        appropriate to use in simple cases where you just need a basic queue with
+        all the default parameters for your message broker.
+
+        :param baseplate: The Baseplate set up for your consumer.
+        :param exchange: The `kombu.Exchange` that you will bind your
+            :py:class:`~kombu.Queue` s to.
+        :param exchange: The `kombu.Connection` to your message broker.
+        :param queue_name: Name for your queue.
+        :param routing_keys: List of routing keys that you will create
+            :py:class:`~kombu.Queue` s to consume from.
+        :param handler_fn: A function that will process an individual message from a queue.
+        :param error_handler_fn: A function that will be called when an error is thrown
+            while executing the `handler_fn`. This function will be responsible for calling
+            `message.ack` or `message.requeue` as it will not be automatically called by
+            `KombuMessageHandler`'s `handle` function.
+        :param health_check_fn: A `baseplate.server.queue_consumer.HealthcheckCallback`
+            function that can be used to customize your health check.
+        :param serializer: A `baseplate.clients.kombu.KombuSerializer` that should
+            be used to decode the messages you are consuming.
+        """
+        queues = []
+        for routing_key in routing_keys:
+            queues.append(kombu.Queue(name=queue_name, exchange=exchange, routing_key=routing_key))
+        return cls(
+            baseplate=baseplate,
+            name=queue_name,
+            connection=connection,
+            queues=queues,
+            handler_fn=handler_fn,
+            error_handler_fn=error_handler_fn,
+            health_check_fn=health_check_fn,
+            serializer=serializer,
+            batch_size=batch_size,
+            batch_timeout=batch_timeout,
+        )
+
+    def build_pump_worker(self, work_queue: BatchWorkQueue) -> KombuBatchConsumerWorker:
```

Comment:
This is just a `queue.Queue` since it's called by the "server" framework, you'll need to initialize the `BatchWorkQueue` in here and pass that to the constructor.